### ISSUE TITLE
On v5.23.05201, the screen reader cannot read aloud

### ISSUE THREAD

**Issue by fltx2004** [BearWare/TeamTalk5#3285]
### Description

I couldn't read aloud using Prism with several screen readers I already have, but the previous beta version was very good

### Application

- [x] qtTeamTalk
- [ ] TeamTalkAndroid
- [ ] iTeamTalk
- [ ] TeamTalkClassic
- [ ] TeamTalkServer

### Platform

- [x] Windows
- [ ] macOS
- [ ] Android
- [ ] iOS
- [ ] Linux

### Expected behavior

screen reader can read aloud normally

### Actual behavior

All calls to prism for reading aloud are invalid

### Steps to reproduce problem

Just change the voice to prism and the voice mode to voice

**Comment by hwangsihu** [comment]
Everything works fine for me. Is it still like that?

**Comment by fltx2004** [comment]
yeah, 5201 is still like this.

**Comment by hwangsihu** [comment]
Hmm, Prism has been updated quite a bit in the meantime. Just in case the update resolves the issue, I will update it. Please test again after merging the PR.
